src/App.tsx

```diff
@@ -1,20 +1,48 @@
 import { useState } from "react";
 import Button from "@mui/material/Button";
 import Login from "./components/Login";
-import { CommonText } from "./common/CustomText";
 import { Header } from "./components/Header/header";
 import { HomePage } from "./components/HomePage/homePage";
+import firstBillboard from "./assets/images/first-billboard-banner.jpg";
+import secondBillboard from "./assets/images/second-billboard-banner.jpg";
+
 import { LoginForm } from "./components/LoginForm/loginForm";
+
 // import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0);
+  const [currentBillboard, setCurrentBillboard] = useState(0);
```

Comment:
since nothing else apart from the homepage is using this state, why not transfer the state into the homePage component?